pipenv-requirements.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
what is this file ?

Reply:
It meant to sync Pipenv with setup.py requires. Now it looks better to just keep setup.py requirements. Removed.